update-data.sh

```diff
@@ -39,6 +42,24 @@ for VER in ${DRIVER_VERSIONS}; do
             NVIDIA_ARCH=x86_64
         fi
 
+        if [ ${ARCH} == aarch64 ]; then
+            if [ ${MAJOR_VER} -lt 470 ]; then
+                continue
+            elif [[ ${TESLA_VERSIONS} == *${VER}* ]]; then
+                continue
+            elif [[ ${VULKAN_VERSIONS} == *${VER}* ]]; then
+                continue
+            else
+                if [ ${VER} == 510.39.01 ]; then
+                    continue
+                elif [ ${VER} == 495.46 ]; then
+                    continue
+                elif [ ${VER} == 470.94 ]; then
+                    continue
+                fi
+            fi
+        fi
```

Comment:
Should all or at least some of these conditions be combined since they all `continue`?